Cauldron/Assets/Scripts/IngredientDroppable.cs

```diff
@@ -186,5 +213,21 @@ public void ChangeHighlight(bool state)
                 isHighlighted = state;
             }
         }
+
+        private async void TrowInCauldron()
+        {
+            if(!useDoubleClick)
+                return;
+            
+            if (cauldron.Mix.Contains(ingredient))
+                 return;
+            
+            const float timeMoveDoubleClick = 1.3f;
+
+            EnableDrag();
+            transform.DOPath(pathDoubleClickAnimation, timeMoveDoubleClick, PathType.CatmullRom);
+            await Task.Delay(1300);
```

Comment:
We can await timeMoveDoubleClick instead of the magic number, OR we can await DoTween by adding ToTask() to it.